docs/standard/base-types/character-encoding/csharp/ConvertToUpper.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
```

Comment:
These files should be hosted in a `snippets` folder, just like `includes` or `images`. Also, should the name of the folder be `character-encoding-introduction` ? So the path is:

```
docs/standard/base-types/snippets/character-encoding-introduction/csharp/*
```